lib/devices_test.go

```diff
@@ -0,0 +1,197 @@
+package usb_tree_lib
```

Comment:
If the tests fail they already say which test failed, seems unnecessarily verbose to repeat that for every message

<img width="872" height="339" alt="image" src="https://github.com/user-attachments/assets/95c01a8c-2d7d-40f5-a482-71820850c3e7" />